### ISSUE TITLE
First release rcX versions

### ISSUE THREAD

**Issue by sirex** [atviriduomenys/spinta#669]
Currently Spinta always releasing stable versions, which are not stable, most of the time.

Instead of releasing stable versions like:
- 0.1.63
- 0.1.64

First release candidates:
- 0.1.64.dev1
- 0.1.64.dev2
- 0.1.64rc1
- 0.1.64rc2
- 0.1.64

In other words, release a testing version before, stable version.

Version numbers should be formatted according to [Version Specifiers](https://packaging.python.org/en/latest/specifications/version-specifiers/#pre-releases) documentation.

Also we need to update documentation, how to install release candidate versions into an isolated version.

Link to documentation, where a new chapter should be added:

https://github.com/atviriduomenys/vadovas/blob/e2a2b793b779ded400ff7af8e84c5c8b9161f7bb/vadovas/spinta.rst?plain=1#L308-L312

New chapter should be added before "Atnaujinimas" and should be named "Testinės aplinkos diegimas".

Documentation should explain how to install release candidate version, as documented here:

- [pip install | Installing pre-release versions](https://pip.pypa.io/en/stable/cli/pip_install/#pre-release-versions)

Also we need to make sure, that all configuration files are isolated from stable version and documentation should recommend following isolation steps:

```
BASEDIR=test/0.1.64
mkdir -p $BASEDIR
python -m venv $BASEDIR/venv
source $BASEDIR/venv/bin/activate
pip install --upgrade --pre spinta
cat $BASEDIR/config.yml
cat > $BASEDIR/config.yml <<EOF
env: production
config_path: $PWD/$BASEDIR
data_path: $PWD/$BASEDIR
default_auth_client: default

keymaps:
  default:
    type: sqlalchemy
    dsn: sqlite:///$PWD/$BASEDIR/keymap.db

backends:
  default:
    type: sql
    dsn: dialect+driver://user:password@host:port/db

manifest: default
manifests:
  default:
    type: csv
    path: $PWD/$BASEDIR/manifest.csv
    backend: default
    keymap: default
    mode: internal

accesslog:
  type: file
  file: $PWD/$BASEDIR/accesslog.json
EOF
export SPINTA_CONFIG=$BASEDIR/config.yml
```

In order to test with all the configuration and data files copy following files:

```
cp -avi ~/.config/spinta $BASEDIR
cp -avi ~/.local/share/spinta $BASEDIR
```

This way files will be preserved.

**Comment by sirex** [comment]
Dokumentacija, dėl testavimo atnaujinta.

https://atviriduomenys.readthedocs.io/spinta.html#testavimas

**Comment by sirex** [comment]
Agreed with the team on version management flow. Created new `0.1` branch:

https://github.com/atviriduomenys/spinta/tree/0.1

All new features will go to `master`, but some bug fixes, related with `0.1` version, will also go to `0.1` branch.

Updated documentation for Spinta developers:

https://spinta.readthedocs.io/en/latest/contrib/git-flow.html

And for Spinta users:

https://atviriduomenys.readthedocs.io/spinta.html#testavimas
